src/buildercore/cfngen.py

```diff
@@ -277,9 +291,10 @@ def _parameterize_gcslogging(gcslogging):
     if context['project']['aws'].get('fastly'):
         backends = context['project']['aws']['fastly'].get('backends', OrderedDict({}))
         context['fastly'] = {
-            'backends': OrderedDict([(n, _parameterize_hostname(b)) for n, b in backends.items()]),
+            'backends': OrderedDict([(n, _build_backend(b)) for n, b in backends.items()]),
             'subdomains': [_build_subdomain(x) for x in context['project']['aws']['fastly']['subdomains']],
             'subdomains-without-dns': [_build_subdomain(x) for x in context['project']['aws']['fastly']['subdomains-without-dns']],
+            'shield': _build_shield(context['project']['aws']['fastly'].get('shield', False)),
```

Comment:
probably needs to be applied to all existing CDN even if `false`, since the Terraform resource definition changes